Devices/kav_efis.device.json

```diff
@@ -16,17 +16,17 @@
   },
   "MessageTypes": [
     {
-      "id": 0,
+      "id": "0",
```

Comment:
The schema said this should be an integer, but all of the code expects it to be a string, so updated the schema and modified the custom device JSON files to use strings for the ID.